examples/upstream-with-k8s-addons/outputs.tf

```diff
@@ -0,0 +1,34 @@
+output "vpc_private_subnet_cidr" {
+  description = "VPC private subnet CIDR"
+  value       = module.vpc.private_subnets_cidr_blocks
+}
+
+output "vpc_public_subnet_cidr" {
+  description = "VPC public subnet CIDR"
+  value       = module.vpc.public_subnets_cidr_blocks
+}
+
+output "vpc_cidr" {
+  description = "VPC CIDR"
+  value       = module.vpc.vpc_cidr_block
+}
+
+output "eks_cluster_id" {
+  description = "EKS cluster ID"
+  value       = module.eks.cluster_id
+}
+
+output "eks_managed_node_groups" {
+  description = "EKS managed node groups"
+  value       = module.eks.eks_managed_node_groups
+}
+
+output "configure_kubectl" {
```

Comment:
I think we can remove all but the kubectl output - do we need these other outputs?

Reply:
> looks good - general question, is `upstream` well known or should we just call it something like `terraform-aws-eks`?

Agree and forgot to call it out, but I first tried to call it `terraform-aws-eks...` but having long name example leads to issues with IRSA/IAM resources names that may be limited to 64 chars :(

Reply:
Tried to keep it consistent with other examples outputs, but will remove it and keep the kubectl output